Octokit/Http/IResponse.cs

```diff
@@ -4,19 +4,50 @@
 
 namespace Octokit
 {
-    public interface IResponse<T> : IResponse
+    /// <summary>
+    /// A response from an API call that includes the deserialized object instance.
+    /// </summary>
+    public interface IApiResponse<out T>
     {
-        new T BodyAsObject { get; set; }
+        /// <summary>
+        /// Object deserialized from the JSON response body.
+        /// </summary>
+        T Body { get; }
+
+        /// <summary>
+        /// The original non-deserialized http response.
+        /// </summary>
+        IResponse HttpResponse { get; }
     }
 
+    /// <summary>
+    /// Represents a generic HTTP response
+    /// </summary>
     public interface IResponse
     {
-        object BodyAsObject { get; set; }
-        string Body { get; set; }
-        Dictionary<string, string> Headers { get; }
-        Uri ResponseUri { get; set; }
```

Comment:
This seems to be the only property that didn't survive the refactor intact. Any particular reasons why?